docs/cli.rst

```diff
@@ -1,11 +1,271 @@
+    list  List available tasks for the current context
+    run   Runs a task
+
+There are even more subcommands available under ``cci task``.
+
+
+
+Getting ``--help``
+------------------
+
+If you want help running a task, type ``cci task run --help``. If you're not certain about what a specific command does, use the ``--help`` option to get more information. 
+
+.. code-block:: console
+
+    $ cci task run --help
+    Usage: cci task run [OPTIONS] TASK_NAME
+
+    Runs a task
+
+    Options:
+    --org TEXT      Specify the target org.  By default, runs against the
+                    current default org
+
+    -o TEXT...      Pass task specific options for the task as '-o option
+                    value'.  You can specify more than one option by using -o
+                    more than once.
+
+    --debug         Drops into pdb, the Python debugger, on an exception
+    --debug-before  Drops into the Python debugger right before task start.
+    --debug-after   Drops into the Python debugger at task completion.
+    --no-prompt     Disables all prompts.  Set for non-interactive mode use such
+                    as calling from scripts or CI systems
+
+    --help          Show this message and exit.    
+
+This gives us a clear usage statement, description, and a list of all options available for use with the command.
+
+
+
+Working with Tasks and Flows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you're just getting started with CumulusCI, don't worry if you aren't sure which of the many tasks and flows to use. We'll show you specific tasks and flows in later sections of the documentation. 
+
+
+Listing Tasks and Flows
+***********************
+
+``cci`` ships with many standard tasks and flows. There are two commmands for listing them:
+
+.. code-block:: console
+
+    $ cci task list
+    $ cci flow list
+
+The tasks and flows listed are specific to the project you're currently in. If you have a custom flow defined in your ``cumulusci.yml`` file for Project A, it only shows if you run ``cci flow list`` in Project A's repository directory.
+
+Tasks and flows are listed by their ``group`` attribute as specified in the ``cumulusci.yml`` file. This means it's easy to edit these groups as you see fit! Any changes made are reflected in the commands.
+
+
+Running Tasks and Flows
+***********************
+When you know the specific task or flow you want to run, execute it with the ``run`` command.
+
+.. code-block:: console
+
+    $ cci task run <name> --org <org> [options]
+    $ cci flow run <name> --org <org> 
+
+This runs the respective task or flow ``<name>`` against the org ``<org>``. (You can see a list of available orgs by running ``cci org list``.)
+
+    Example: The ``run_tests`` task executes Apex unit tests. If you have an org called ``dev``, you can run this task against it with the command ``cci task run run_tests --org dev``.
+
+Tasks usually require additional options to be passed when using the ``cci task run`` command.
```

Comment:
```suggestion
Tasks usually require additional options to be passed when using the ``cci task run`` command. Learn about how to find information about task options in :ref:`Task Info & Options`.
```

Reply:
I wouldn't worry about making these type of changes currently. This sounds like something that could be shored up after the initial docs release.